translate/src/modules/search/components/SearchPanel.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useCallback, useState } from 'react';
+import { Localized } from '@fluent/react';
+import classNames from 'classnames';
+import { SEARCH_OPTIONS } from '../constants';
+import type { SearchState, SearchType } from './SearchBox';
+
+import { useOnDiscard } from '~/utils';
+
+import './SearchPanel.css';
+
+type Props = {
+  options: SearchState;
+  applyOptions: () => void;
+  toggleOption: (value: string, filter: SearchType) => void;
```

Comment:
There are still a few instances of `filter` and `SearchFilter` in this file. Let's use searchOptions.